README.md

```diff
@@ -19,6 +19,24 @@ source 'https://github.com/CocoaPods/Specs.git'
 [2] add use_frameworks! commend to your target
 target 'YOUR_TARGET' do
   use_frameworks!
+  
+[3] At the end of your code please add 
+
+  post_install do |installer|
+   react_native_post_install(
+     installer,
+     :mac_catalyst_enabled => false
+   )
+   installer.pods_project.targets.each do |target|
+     target.build_configurations.each do |config|
+       config.build_settings['BUILD_LIBRARY_FOR_DISTRIBUTION'] = 'YES'
+       config.build_settings['EXCLUDED_ARCHS[sdk=iphonesimulator*]'] = 'arm64'
+     end
+   end
+   __apply_Xcode_12_5_M1_post_install_workaround(installer)
+ end
+end
+
 ```
 
 3. Run `NO_FLIPPER=1 pod install` to install the necessary pods.
```

Comment:
@SencyNetanel what about this? in the instructions you sent me in Saturday it was simply 'pod install'
so we still need to tell the clients to run NO_FLIPPER=1 ?

Reply:
okay got it